README.md

```diff
@@ -1,14 +1,33 @@
-# Compute@Edge default template for Rust
+# Compute@Edge default starter kit for Rust
 
-Default package template used by the [Fastly CLI][cli] for Rust-based Compute@Edge projects.
+Get to know the Fastly Compute@Edge environment with a basic starter that demonstrates routing, simple synthetic responses, and overriding caching rules.
 
-## Usage
-- Install the [Fastly CLI][latest] to your `$PATH`
-- Run `fastly compute init`
+**For more details about this and other starter kits for Compute@Edge, see the [Fastly developer hub](https://developer.fastly.com/solutions/starters)**
```

Comment:
This URL currently 404's, i'm assuming as the content hasn't been launched yet? Should we not merge this PR until said content is live?

Reply:
This is a chicken/egg problem.  We could either not care too much and just try to co-ordinate the merges to be close together, or we could split it up so that this line is added to the readme in a subsequent PR.

Reply:
I quick co-ordination of merges should be sufficient :+1: